### PR TITLE
Create new grid listing as breakpoint has changed on About profiles

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

**Comment by discodavey** [comment]
We have had to create a new grid listing for the About profiles from the grid-listing class as the breakpoint is needed at 480 rather than 730px.

File: assets/sass/patterns/organisms/about-profiles.scss [hunk 1]
```diff
@@ -1,14 +1,63 @@
...
 }
 
+.grid-listing__about {
+  @media only all and (min-width: #{get-rem-from-px($bkpt-site--small)}em) {
+    grid-template-columns: repeat(2, 1fr);
+    grid-column-gap: 2.25rem;
+    grid-row-gap: #{get-rem-from-px($blg-space-smallish-in-px)}rem;
+    justify-content: center;
+
+    // Supporting clients don't need clearfix
```
Comment (nlisgo) [inline assets/sass/patterns/organisms/about-profiles.scss:43]: Is this meant to say, "Supporting clients that don't need clearfix"?

File: assets/js/components/AboutProfiles.js [hunk 2]
```diff
@@ -11,11 +11,8 @@ module.exports = class AboutProfiles {
     this.window = _window;
     this.doc = doc;
 
-    this.$elm.classList.add('grid-listing');
+    this.$elm.classList.add('grid-listing__about');
```
Comment (nlisgo) [inline assets/js/components/AboutProfiles.js:14]: are there styles that we can borrow from `grid-listing` too and this can be a variant?

Comment (discodavey) [reply assets/js/components/AboutProfiles.js:14]: I think I have used all I can apart from the breakpoint. I will have another look this morning.

Comment (discodavey) [reply assets/js/components/AboutProfiles.js:14]: I have created three mixins so these can be used between both grid-listing and grid-listing__about.

**Comment by discodavey** [comment]
Closing as no longer needed.

**Pull request closed without merging**